### PR TITLE
DISCOVERY-43 - support for ansible automation controller

### PR DESCRIPTION
**Pull request by bruno-fs**
Add support to new "ansible" data source.

### CODE REVIEW

**Review by abellotti — APPROVED** [review]

**Comment by abellotti** [comment]
LGTM!! 👍

**Review by nicolearagao — APPROVED** [review]
LGTM 👍🏼

**Review by infinitewarp — APPROVED** [review]

**Review by ruda — CHANGES_REQUESTED** [review]
Hope you like this suggestions to update the help messages.

File: qpc/cred/add.py [hunk 1]
```diff
@@ -48,12 +49,7 @@ def __init__(self, subparsers):
...
             dest="type",
-            choices=[
-                credential.NETWORK_CRED_TYPE,
-                credential.VCENTER_CRED_TYPE,
-                credential.SATELLITE_CRED_TYPE,
-                credential.OPENSHIFT_CRED_TYPE,
-            ],
+            choices=SOURCE_TYPE_CHOICES,
             metavar="TYPE",
             help=_(messages.CRED_TYPE_HELP),
```
[the remark this answers is not in the document]
Comment (nicolearagao) [reply qpc/cred/add.py:54]: Hey @ruda, we will address this soon. We have a jira ticket open for this issue: https://issues.redhat.com/browse/DISCOVERY-211. We will create a new one to make sure to include info about Ansible as well.

[the remark this answers is not in the document]
Comment (bruno-fs) [reply qpc/cred/add.py:54]: DISCOVERY-211 covers only manpage, not help text (and is ocp related).

I'm aligned with rudá on this one, it's better if we have it done.

[the remark this answers is not in the document]
Comment (nicolearagao) [reply qpc/cred/add.py:54]: I see, thanks for clarifying. I thought an overall update was covered by that issue.

[the remark this answers is not in the document]
Comment (bruno-fs) [reply qpc/cred/add.py:54]: done.
notice I'm introducing a breaking change to the "OCP_TOKEN" message (should not be married to OCP anyways. nothing holds us from using in another integration in the future   (satellite and ansible do have support for that)

**Review by ruda — APPROVED** [review]
Perfect! LGTM.

**Pull request merged**